src/main/java/org/example/repository/QuestionRepositoryImpl.java

```diff
@@ -0,0 +1,99 @@
+package org.example.repository;
+
+import org.example.model.Question;
+import org.example.repository.dao.QuestionRepository;
+import java.sql.*;
+
+public class QuestionRepositoryImpl implements QuestionRepository {
+
+    private final Connection connection;
```

Comment:
you can use singleton pattern  ( discased on the lesson ) instead of saving connection as a class attribute